src/reportWebVitals.ts

```diff
@@ -0,0 +1,15 @@
+import { ReportHandler } from 'web-vitals'
+
+const reportWebVitals = (onPerfEntry?: ReportHandler) => {
+  if (onPerfEntry && onPerfEntry instanceof Function) {
+    import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
+      getCLS(onPerfEntry)
```

Comment:
Can you add a comment to explain what these abbreviations stand for or at least where I can look up what they mean?

Reply:
@waleedkadous  This a measuring performance tool, you can have a look at it for more details https://create-react-app.dev/docs/measuring-performance/